File: rs.hmi.vueclient/src/Models/WebAPI/RectModel.ts
/**
 * 矩形框
 */
export class RectModel {
  public Left: number;
  public Top: number;
  public Width: number;
  public Height: number;
  constructor(left: number, top: number, width: number, height: number) {
    this.Left = left;
    this.Top = top;
    this.Width = width;
    this.Height = height;
  }
}

File: RS.Models/RectModel.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RS.Models
{
    /// <summary>
    /// 矩形框
    /// </summary>
    public class RectModel
    {
        public RectModel(double left, double top, double width, double height)
        {
            this.Left = left;
            this.Top = top;
            this.Width = width;
            this.Height = height;
        }

        public double Left { get; set; }
        public double Top { get; set; }
        public double Width { get; set; }
        public double Height { get; set; }

    }
}
